server-launcher/pom.xml

```diff
@@ -35,45 +35,9 @@
     <dependencies>
         <dependency>
             <groupId>javax.servlet</groupId>
-            <artifactId>servlet-api</artifactId>
-            <version>2.5</version>
+            <artifactId>javax.servlet-api</artifactId>
+            <version>3.1.0</version>
```

Comment:
I assume that this version of servlet API works with Jetty 6 as well. However, that is a bit of a change from the older one, so we should probably perf test this well.